.gitignore

```diff
@@ -136,10 +139,10 @@ dist
 
 ### VisualStudioCode ###
 .vscode/*
-# !.vscode/settings.json
-# !.vscode/tasks.json
-# !.vscode/launch.json
-# !.vscode/extensions.json
+!.vscode/settings.json
```

Comment:
Why are we adding some of vscode setting files to the repository? Is it cause we want all vscode users to have the same settings, extensions, etc?

Reply:
> Smart IDEs (such as VSCode or IntelliJ) require special configuration for TypeScript to work.

These files have settings made by the Yarn Editor SDK.
https://yarnpkg.com/advanced/editor-sdks